modules/nw-cudn-cr.adoc

```diff
@@ -0,0 +1,146 @@
+//module included in the following assembly:
+//
+// *networking/multiple_networks/understanding-user-defined-networks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-cudn-cr_{context}"]
+= Creating a ClusterUserDefinedNetwork custom resource
+
+The following procedure creates a cluster-wide user-defined network. Based upon your use case, create your request using either the `cluster-layer-two-udn.yaml` example for a `Layer2` topology type or the `cluster-layer-three-udn.yaml` example for a `Layer3` topology type.
+
+//We won't have these pieces till GA in 4.18.
+//[NOTE]
+//====
+//If any cluster default networked pods exist before the user-defined network is created, any further pods created in this namespace will return an error message: `What_is_this`.
+//====
+
+.Procedure
+
+. Create a request for either a `Layer2` or `Layer3` topology type cluster-wide user-defined network:
+
+.. Create a YAML file, such as `cluster-layer-two-udn.yaml`, to define your request for a `Layer2` topology as in the following example:
++
+[source, yaml]
+----
+apiVersion: k8s.ovn.org/v1
+kind: ClusterUserDefinedNetwork
+metadata:
+  name: <cudn_name> # <1>
+spec:
+  namespaceSelector: # <2>
+    matchLabels: { "<example_namespace_one>": "<example_namespace_two>" }
+  network:
+    topology: Layer2 # <3>
+    layer2: <4>
+      role: Primary # <5>
+      subnets:
+        - "2001:db8::/64"
+        - "10.100.0.0/16" # <6>
+----
+<1> Name of your `ClusterUserDefinedNetwork` resource. This should not be `default` or duplicate of any resource created by the Cluster Network Operator (CNO).
```

Comment:
Asking to understand, the `ClusterUserDefinedNetwork` is not managed by CNO, why do we have this restriction?
I dont recall it being validated or prevented by any controller.

Reply:
Copied from UDN docs. I wasn't sure which callouts (eg <1>, <2>) from those docs would apply to CUDN, and which wouldn't. I have updated.